java/arcs/core/util/Scheduler.kt

```diff
@@ -73,7 +75,9 @@ class Scheduler(
     }
 
     /** Wait for the [Scheduler] to become idle. */
+    /* internal */
     suspend fun waitForIdle() {
+        delay(100)
```

Comment:
Where does this number come from, and could it be a source of flakyness if it is too short?

Reply:
It could be.. I think I want to change away from "waitForIdle" in general, since it's typically used in place of something where we could instead wait for an actual callback to happen.